specs/notation-config.md

```diff
@@ -0,0 +1,59 @@
+# Notation Configuration
+
+To enable persisted configuration, simplifying the execution of the `notation` cli, the following configuration file will be available
+
+> Note: there will be a policy based configuration that will come at a later point.
+
+## Location
+
+The default location and file will be stored at: `~/.notary/notation-config.json`. The `notation` cli and libraries will support alternate locations through a `config-location` parameters.
```

Comment:
Will `~/.notation/config.json` be better?

Reply:
`notation` is one of the cli's, but the configuration would be for notary. I'm trying to avoid having too much folder sprawl.

Reply:
For now I feel we can go with  @shizhMSFT's recommendation and if we want to move the default location or file name we can track that as a separate item. The change currently will only affect directories that are typical locations for this specific CLI.